client/src/components/ReviewsShowPage.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from "react";
+
+const ReviewsShowPage = ({ id }) => {
+  const [reviewList, setReviewList] = useState([]);
+
+  const getReviews = async () => {
```

Comment:
the reviews should be queried when we fetch for the activity on this same webpage (the parent component) - that way there are not 2 fetch requests in the same page for essentially the same info